src/components/InstanceMonitoring/InstanceMonitoringCard.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Icon } from 'carbon-components-react';
+import { InstanceMonitoringContent } from './InstanceMonitoringContent';
+
+export class InstanceMonitoringCard extends Component {
+  render() {
```

Comment:
Given this class does not seem to have any state, does it make sense to make this stateless? Same for other classes created in this PR.